plugins/git/README.md

```diff
@@ -102,14 +103,15 @@ plugins=(... git)
 | glod                 | git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset'                           |
 | glods                | git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%ad) %C(bold blue)<%an>%Creset' --date=short              |
 | glola                | git log --graph --pretty='%Cred%h%Creset -%C(auto)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --all                     |
+| glom                 | git log --oneline --decorate --color $(git_main_branch)                                                                          | List commit since the main branch |
 | glog                 | git log --oneline --decorate --graph                                                                                             |
 | gloga                | git log --oneline --decorate --graph --all                                                                                       |
 | glp                  | git log --pretty=\<format\>                                                                                                      |
 | gm                   | git merge                                                                                                                        |
-| gmom                 | git merge origin/$(git_main_branch)                                                                                                            |
+| gmom                 | git merge origin/$(git_main_branch)                                                                                              |
```

Comment:
Trailing whitespace

Reply:
I'm using vim with trailing whitespace highlighted and I can't see any:
![Capture d’écran 2021-06-15 à 11 20 05](https://user-images.githubusercontent.com/879889/122027824-bbef2800-cdcb-11eb-8458-95519fddacdd.png)

Am I misunderstanding something?

Reply:
Hmmm.... I see dots on the GitHub "files changed" tab, which for my config means whitespace. I can even highlight it...

![image](https://user-images.githubusercontent.com/52163910/122056372-86404400-cdb7-11eb-898d-681c472c5eb7.png)

happens in incognito too

![image](https://user-images.githubusercontent.com/52163910/122056444-96582380-cdb7-11eb-9794-17d20816c337.png)

Reply:
It is normal you have trailing space after `git merge upstream/$(git_main_branch)` because it is followed by about 90 whitespace and the final `|` of the table.

Reply:
Oh, that makes sense. Sorry.